packages/react/accordion/src/Accordion.tsx

```diff
@@ -154,22 +162,32 @@ const AccordionMultiple = React.forwardRef((props, forwardedRef) => {
   );
 
   const handleItemClose = React.useCallback(
-    (itemValue) => setValue((prevValue = []) => prevValue.filter((value) => value !== itemValue)),
-    [setValue]
+    (itemValue) =>
+      setValue((prevValue = []) => {
+        if (preventAllClosed && prevValue.length === 1 && prevValue[0] === itemValue)
+          return prevValue;
```

Comment:
This is the bit where the `preventAllClosed` is implemented for `type="multiple"`.